File: 백준/Bronze/2455. 지능형 기차/지능형 기차.py
on, off = map(int, input().split(" "))
one = -1 * on + off;
on, off = map(int, input().split(" "))
two = one + -1 * on + off;
on, off = map(int, input().split(" "))
three = two + -1 * on + off;
on, off = map(int, input().split(" "))
four = three + -1 * on + off;

if(one > two and one >  three and one > four):
    print(one);
elif(two > one and two >  three and two > four):
    print(two);
elif(three > two and three >  one and three > four):
    print(three);
elif(four > two and four >  three and four > one):
    print(four);

File: 백준/Bronze/2455. 지능형 기차/지능형 기차.cc
#include <stdio.h>

int main(){
    int on, off, one, two, three, four;
    scanf("%d %d", &on, &off);
    one = -1 * on + off;
    scanf("%d %d", &on, &off);
    two = one + -1 * on + off;
    scanf("%d %d", &on, &off);
    three = two + -1 * on + off;
    scanf("%d %d", &on, &off);
    four = three + -1 * on + off;
    
    if(one > two && one >  three && one > four){
        printf("%d", one);
    }else if(two > one && two >  three && two > four){
        printf("%d", two);
    }else if(three > two && three >  one && three > four){
        printf("%d", three);
    }else if(four > two && four >  three && four > one){
        printf("%d", four);
    }
}